File: processor/src/main/java/com/kazakago/preferhythm/constants/Annotations.java
package com.kazakago.preferhythm.constants;

import com.squareup.javapoet.AnnotationSpec;

/**
 * Annotation Constants
 *
 * Created by KazaKago on 2017/03/14.
 */
public class Annotations {

    public static final AnnotationSpec NonNull = AnnotationSpec.builder(Types.NonNull).build();

    public static final AnnotationSpec Nullable = AnnotationSpec.builder(Types.Nullable).build();

}

File: processor/src/main/java/com/kazakago/preferhythm/constants/Annotations.kt
package com.kazakago.preferhythm.constants

import com.squareup.javapoet.AnnotationSpec

/**
 * Annotation Constants
 *
 * Created by KazaKago on 2017/03/14.
 */
class Annotations {

    companion object {
        @JvmStatic
        val NonNull = AnnotationSpec.builder(Types.NonNull).build()!!

        @JvmStatic
        val Nullable = AnnotationSpec.builder(Types.Nullable).build()!!
    }

}
